_pages/address/15-address_confirmed.html

```diff
@@ -7,11 +7,16 @@
 <div class="container sm-pt3">
   <div class="px2 py3 sm-py5 sm-px6 mx-auto sm-mb5 border-box card">
     <form autocomplete="off" role="form" class="simple_form new_profile mb5" id="new_profile" action="{{ "/settings/recovery-code/" | prepend: site.baseurl }}" accept-charset="UTF-8">
-      <img alt="images" class="mb2" width="200" src="{{ "/images/addy-confirm@3x.png" | prepend: site.baseurl }}">
-      <h1 class="h3 mt2 mb0">
-        We have confirmed your address.
+      <img alt="images" class="mb2" width="124" src="{{ "/images/LOA3-success.svg" | prepend: site.baseurl }}">
+      <h1 class="h3 mt2 mb3">
+        We have confirmed your identity!
       </h1>
-      <p class="mb3">All that's left is making sure you, and only you, have access to your account, even if you lose your phone.</p>
+      <div class="pb-tiny bg-green col-2">
+      </div>
+      <h1 class="h3 mt3 mb0">
+        One last step.
+      </h1>
```

Comment:
Should this second `H1` be an `H2`

Reply:
great catch and a reminder to me to stay focused on proper semantic html. BUT since this will never see the light of day as a public facing site, I'm fine leaving as is. Onward!